app/views/layouts/_footer.haml

```diff
@@ -6,7 +6,7 @@
           %h3= t('social_title')
           %ul.list-inline
             %li
-              %a.btn-social.btn-outline{:href => "https://twitter.com/BastienBarbe"}
+              %a.btn-social.btn-outline{:href => "https://twitter.com/intent/follow?screen_name=BastienBARBE"}
```

Comment:
Line is too long. [110/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace